diff --git a/docker-images/kafka-based/kafka/Makefile b/docker-images/kafka-based/kafka/Makefile
index d5fde4588..c23f9112d 100644
--- a/docker-images/kafka-based/kafka/Makefile
+++ b/docker-images/kafka-based/kafka/Makefile
@@ -42,6 +42,8 @@ clean:
 .kafka-$(KAFKA_VERSION).tmp: ../../artifacts/binaries/kafka/archives/kafka_2.13-$(KAFKA_VERSION).tgz
 	test -d tmp/$(KAFKA_VERSION) || mkdir -p tmp/$(KAFKA_VERSION)
 	tar xvfz ../../artifacts/binaries/kafka/archives/kafka_2.13-$(KAFKA_VERSION).tgz -C tmp/$(KAFKA_VERSION) --strip-components=1
+	test -f tmp/$(KAFKA_VERSION)/libs/rocksdbjni-6.22.1.1.jar && cp -f $(HOME)/$(ROCKSDBJNI_DIR)/rocksdbjni-6.22.1.1.jar tmp/$(KAFKA_VERSION)/libs/ || true
+	test -f tmp/$(KAFKA_VERSION)/libs/rocksdbjni-6.19.3.jar && cp -f $(HOME)/$(ROCKSDBJNI_DIR)/rocksdbjni-6.19.3.jar tmp/$(KAFKA_VERSION)/libs/ || true
 	touch .kafka-$(KAFKA_VERSION).tmp
 
 .kafka-$(KAFKA_VERSION)-ignorelist.tmp: ../../artifacts/binaries/kafka/$(KAFKA_VERSION).ignorelist
